MyCryptoHQ/MyCrypto
# Feature/goerli network

### Description

Implement MyCrypto Goerli network and Etherscan Goerli network.

### Changes

Added Goerli as a default network option.

### Steps to Test

Switch to Goerli and ensure network works as expected.

[MyCryptoBuilds](https://mycryptobuilds.com/fe0804389fc039e10862c5f902be52b4a0ff0780)
